nservicebus/upgrades/absdatabus-4to5.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Azure Blob Storage Databus Upgrade Version 4 to 5
+reviewed: 2020-10-29
+component: ABSDataBus
+related:
+ - nservicebus/upgrades/7to8
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 8
+---
+
+Upgrade from version 7 to 8 does not require any code changes.
```

Comment:
```suggestion
Upgrade from version 4 to 5 does not require any code changes.
```

although that is probably soon no longer true

Reply:
Are you suggesting to drop the cleanup in 5 but not 4?

Reply:
Deprecate with error in 4 and remove in 5.